src/utils/common.js

```diff
@@ -231,32 +234,59 @@ export const getiOSNavbarHeight = (): number => {
   return 0;
 };
 
+const DEFAULT_TRANSITION_SCREENS = [MANAGE_USERS_FLOW];
+
+const getIfNeedsDefTransition = (transitionProps: TransitionProps, prevTransitionProps: TransitionProps) => {
+  return DEFAULT_TRANSITION_SCREENS.some(
+    screenName =>
+      screenName === transitionProps.scene.route.routeName ||
+      (prevTransitionProps && screenName === prevTransitionProps.scene.route.routeName),
+  );
+};
+
+const getTransitionDuration = (isFaster: boolean) => {
+  let duration = 400;
+  if (isFaster) {
+    duration = Platform.OS === 'ios' ? 500 : 250;
+  }
+  return duration;
+};
+
+const getTransitionSpec = (isFasterAnimation: boolean) => {
```

Comment:
Minor: either there is no need to wrap return with `(...)` or you can just remove `{ return`